test/e2e/framework.go

```diff
@@ -1156,16 +1150,16 @@ func (data *TestData) deletePodAndWait(timeout time.Duration, name string, ns st
 		return false, nil
 	})
 	if err == wait.ErrWaitTimeout {
-		return fmt.Errorf("Pod '%s' still visible to client after %v", name, timeout)
+		return fmt.Errorf("pod '%s' still visible to client after %v", name, timeout)
```

Comment:
why change `Pod` to `pod`? typo?

Reply:
`Error string should not be capitalized or end with punctuation mark`

Reply:
It actually can be upper case when it's nouns https://github.com/golang/go/wiki/CodeReviewComments#error-strings

Reply:
Okay, I changed it back. Just so you know that there are a couple of other places in the e2e package where instances like `Pod` are not capitalized in the e2e package. Might need to follow up on that.